contents/docs/integrations/js-integration.md

```diff
@@ -162,6 +162,8 @@ Opt a user back in:
 posthog.opt_in_capturing();
 ```
 
+**Note:** This method is a suggested way to prevent capturing data from the admin on the page. A simple way to do this is to access the page as the admin (or any other user on your team you wish to stop capturing data on), and call `posthog.opt_out_capturing();` directly on the developer console. 
```

Comment:
I'd add something like "... or call it directly in your app after an admin logs in."

Reply:
Done!